### PR TITLE
Add user lang in geocoder query + cleanup

### PR DESCRIPTION
**Pull request by amatissart**
* Add user lang in geocoder query
* Some cleanup in dependencies and test configuration
* Use stretch-slim as (smaller) base image in Dockerfile

### CODE REVIEW

File: tests/integration/tools.js [hunk 1]
```diff
@@ -37,6 +37,9 @@ export function store(page, soreKey,  data) {
 }
 
 export const clearStore = async function(page) {
+  if(page.url() === 'about:blank') {
```
Comment (Julien-laville) [inline tests/integration/tools.js]: Nice

File: tests/integration/tests/favorite.js [hunk 2]
```diff
@@ -17,11 +17,11 @@ beforeAll(async () => {
 test('toggle favorite panel', async () => {
   expect.assertions(2)
   await page.goto(APP_URL)
-  let favPanelHidden = await page.waitForSelector(".favorites_panel--hidden")
-  expect(favPanelHidden).not.toBeFalsy()
+  let favPanelHidden = await page.waitForSelector(".favorites_panel--hidden", {hidden : true})
```
Comment (Julien-laville) [inline tests/integration/tests/favorite.js]: Thank's, a lot more clear now

File: tests/integration/helpers/response_handler.js [hunk 3]
```diff
@@ -5,6 +5,12 @@ export default class ResponseHandler {
     this.page = page
   }
 
+  static async init(page) {
```
Comment (Julien-laville) [inline tests/integration/helpers/response_handler.js]: ok

File: bin/app.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ const app = express()
 const logger = bunyan.createLogger({
     name: 'erdapfel',
     stream: process.stdout,
-    level: 'info',
+    level: process.env.NODE_ENV === 'test' ? 'warn' : 'info',
```
Comment (Julien-laville) [inline bin/app.js]: ok

File: config/default_config.yml [hunk 5]
```diff
@@ -15,6 +15,7 @@ server:
 services:
   geocoder:
     url: override_by_environment
+    useLang: true
```
Comment (Julien-laville) [inline config/default_config.yml]: is it the default value we want ?

File: tests/integration/tests/autocomplete.js [hunk 6]
```diff
@@ -42,6 +41,24 @@ test('search and clear', async () => {
   expect(searchValueAfterClear).toEqual('')
 })
 
+test('search has lang in query', async () => {
+  const langPage = await browser.newPage()
+  await langPage.setDefaultTimeout(2000)
```
Comment (Julien-laville) [inline tests/integration/tests/autocomplete.js]: is it stil required ?

File: config/default_config.yml (hunk 5 above)
Comment (amatissart) [reply config/default_config.yml]: I think so. It has no impact for now, and will be helpful as soon as the geocoder is released.

File: tests/integration/tests/autocomplete.js (hunk 6 above)
Comment (amatissart) [reply tests/integration/tests/autocomplete.js]: Not required. But I would keep it, as it could help with debugging in the future....

Comment (amatissart) [reply tests/integration/tests/autocomplete.js]: I will add a comment about it.

**Pull request merged**